kale/predict/losses.py

```diff
@@ -27,6 +27,72 @@ def cross_entropy_logits(linear_output, label, weights=None):
     return loss, correct
 
 
+def topk_accuracy(output, target, topk=(1,)):
+    """Computes the precision@k for the specified values of k
+
+    Args:
+        output (Tensor): Generated predictions. Shape: [batch_size, class_count].
+        target (Tensor): Ground truth. Shape: [batch_size]
+        topk (tuple(int)): Compute accuracy at top-k for the values of k specified in this parameter.
+    Returns:
+        list(Tensor): A list of tensors of the same length as topk with boolean in.
+
+    Examples:
+        output = torch.tensor([0.3, 0.2, 0.1], [0.3, 0.2, 0.1])
+        target = torch.tensor((0, 2))
+        top1, top2 = topk_accuracy(output, target, topk=(1, 2))
+    """
+
+    maxk = max(topk)
+    # batch_size = target.size(0)
```

Comment:
What is the purpose of this commented line? If it is a comment, write in plain text to explain. Otherwise, it is confusing to code readers. Likewise for other commented code below.